Dockerfile.plugins

```diff
@@ -1,21 +1,38 @@
+# Build the plugin
 FROM node:16.4 as builder
 
 WORKDIR /headlamp-plugins
 
+# Copy the example plugins source code to the working directory
 COPY ./plugins/examples /headlamp-plugins/
 
+# Create a build directory
 RUN mkdir -p /headlamp-plugins/build
 
-# Build the plugin
+# Copy the plugin source code to the working directory
+COPY ./plugins/headlamp-plugin /headlamp-plugins/headlamp-plugin
+
+# Change to the headlamp-plugin directory, install dependencies,
+# and link the local plugin to the global headlamp-plugin, so that
+# we can use the headlamp-plugin to build the example plugins
+# with the latest changes.
```

Comment:
comment mentions link, please update it otherwise good to go.

Reply:
Edited and updated PR.